admin/modules/user/users.php

```diff
@@ -992,7 +992,7 @@
 	}
 	else
 	{
-		$user['avatar'] = "../".$mybb->settings['useravatar'];
+		$user['avatar'] = "../".str_replace('{theme}', 'images', $mybb->settings['useravatar']);
```

Comment:
But that's going to show a 404 in the Users in ACP. That is going to give ../images/images/default_avatar.png